modules/cli/src/main/scala/scala/cli/commands/SharedCompilationServerOptions.scala

```diff
@@ -185,26 +185,28 @@ final case class SharedCompilationServerOptions(
         BloopVersion(Constants.bloopVersion)
       ))(v => BloopRifleConfig.Strict(BloopVersion(v)))
 
-  def bloopDefaultJvmOptions(logger: Logger): List[String] = {
```

Comment:
Why don't we use the `logger` to print stuff? (I saw there were some `println`s before already, but I'm not sure why they're here…)

Reply:
Right, a logger should be used